### PR TITLE
[EM-164] Add external contributions through admin

### PR DESCRIPTION
**Pull request by SandroDamilano**
## What does this PR do?
It adds External Pull Requests to the admin to be able to create new ones manually

Screen-Recording-2020-10-30-at-1

Resolves [#164](https://rootstrap.atlassian.net/browse/EM-164)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -5,9 +5,23 @@ GITHUB_ANALYZER_PASSWORD=postgres
...
 
+# Own Github organization
+GITHUB_ORGANIZATION=rootstrap
+
+# Worpress data
+WORDPRESS_SITE_ID=
+WORDPRESS_CLIENT_ID=
+WORDPRESS_CLIENT_SECRET=
+WORDPRESS_USERNAME=
+WORDPRESS_PASSWORD=
```
Comment (horacio) [inline .env.example:16]: Are these variables related to this particular PR?

File: app/models/external_pull_request.rb [hunk 2]
```diff
@@ -26,6 +27,8 @@
 
 class ExternalPullRequest < ApplicationRecord
   belongs_to :external_project
+  alias project external_project
```
Comment (horacio) [inline app/models/external_pull_request.rb:30]: I think this would work for read-only ops, but what about assignments? Please, disregard this if it isn't something of importance. I'm bringing it up just in case.

File: db/structure.sql [hunk 3]
```diff
@@ -572,6 +573,7 @@ CREATE TABLE public.external_pull_requests (
     external_project_id bigint NOT NULL,
     created_at timestamp(6) without time zone NOT NULL,
     updated_at timestamp(6) without time zone NOT NULL,
+    number integer,
```
Comment (horacio) [inline db/structure.sql:576]: Just out of curiosity: why aren't we adding a NOT NULL constraint here?

**Review by horacio — APPROVED** [review]
Looking good, Sandro! 👏  I left some non-blocking comments, mostly doubts. I checked out this branch and tried locally, works like a charm.

**Review by santiagovidal — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (SandroDamilano) [reply .env.example:16]: No, they are not! I think I originally added these for when someone new would join the team, so that they could have the correct env vars. But well, I finished a bit too late 😅 
In any case, I could totally add these changes in another PR

File: app/models/external_pull_request.rb (hunk 2 above)
Comment (SandroDamilano) [reply app/models/external_pull_request.rb:30]: Yeah, it probably doesn't work for assignments, but I just needed it for read operations anyway (to make `ExternalPullRequest` polymorphic with `PullRequest` for the `project` message).
Do you think it would be best for it to work for assignments as well?

File: db/structure.sql (hunk 3 above)
Comment (SandroDamilano) [reply db/structure.sql:576]: Just so that we didn't need to add a default value for old records (this way, old records will have the `number` attribute in `nil` and not some random number that could be misleading).
If we made a backfill, then we could add the constraint

File: .env.example (hunk 1 above)
Comment (horacio) [reply .env.example:16]: Personally I think it's fine

File: app/models/external_pull_request.rb (hunk 2 above)
Comment (horacio) [reply app/models/external_pull_request.rb:30]: If it isn't something in our short-term backlog, then I would leave it as it is and tackle it when it becomes a problem.

File: db/structure.sql (hunk 3 above)
Comment (horacio) [reply db/structure.sql:576]: Ahhhh I see. Nice!

**Review by nicofh — APPROVED** [review]
